apps/dash-object-detection/README.md

```diff
@@ -0,0 +1,74 @@
+# Object Detection
+
+This is a demo of the Dash interactive Python framework developed by [Plotly](https://plot.ly/).
+
+Dash abstracts away all of the technologies and protocols required to build an interactive web-based application and is a simple and effective way to bind a user interface around your Python code. To learn more check out our [documentation](https://plot.ly/dash).
+
+Try out the [demo app here](https://dash-gallery.plotly.host/dash-object-detection/).
+
+![Animated1](screenshots/dash-object-detection-demo.gif)
+
+## Getting Started
+
+### Using the demo
+
+To get started, select a footage you want to view, and choose the display mode (with or without bounding boxes). Then, you can start playing the video, and the visualization will be displayed depending on the current time.
+
+### Running the app locally
+
+First create a virtual environment with conda or venv inside a temp folder, then activate it.
+
+```
+virtualenv dash-object-detection
+
+# Windows
+dash-object-detection\Scripts\activate
+# Or Linux
+source venv/bin/activate
+```
+
+Clone the git repository, and install the requirements with pip
+
+```
+git clone https://github.com/plotly/dash-sample-apps.git
+cd dash-sample-apps/apps/dash-object-detection
+pip install -r requirements.txt
+```
+
+Run the app
+
+```
+python app.py
+```
+
+## About this app
+
+The videos are displayed using a community-maintained Dash video component. It is made by two Plotly community contributors. You can find the [source code here](https://github.com/SkyRatInd/Video-Engine-Dash).
```

Comment:
Is this still the correct link? Aren't we using Dash Player?

Reply:
This should be directed to the latest Dash player component(https://github.com/plotly/dash-player),source code from community contributor was outdated for over a year and we could quote them in dash-player repo.